src/dialogCustomFields.html

```diff
@@ -84,7 +84,7 @@
           chkboxInput.setAttribute('value', JSON.stringify(field));
           chkboxLabel.setAttribute('for', 'customfield-id-'+field.key);
           chkboxLabel.innerHTML = field.name;
-          if (!field.supported) chkboxLabel.innerHTML += ' <small><em>(Type: ' + JSON.stringify(field.type) + (field.supported?'':' - not supported') + ')</em></small>';
+          if (!field.supported) chkboxLabel.innerHTML += ' <small><em>(Type: ' + JSON.stringify(field.schemaType) + (field.supported?'':' - not supported') + ')</em></small>';
```

Comment:
The rest of the code when fetching fields from JIRA uses schemaType as the property name so I changed it here so I could re-use the common code.